src/tests/libecalc/input/mappers/variables_mapper/test_timeseries.py

```diff
@@ -152,73 +153,90 @@ def test_valid_time_series_unsorted(self):
         (
             ["DATE", "OIL_PROD", "BVBV"],
             [["01.01.2017"], [5016]],
-            "The number of columns provided do not match for header and data: data: 1, headers: 2",
-        ),
-        # headers, data mismatch (-1)
-        (
-            ["DATE", "OIL_PROD"],
-            [["01.01.2017"], [5016], [232]],
-            "The number of columns provided do not match for header and data: data: 2, headers: 1",
+            snapshot("""\
```

Comment:
Removed test. Current behavior uses headers as what exists, then look up columns based on that. If we are going to catch this we should validate the resource. 

If this is a case where the user forgot to add the header, it should come up when the column is referenced.